src/asmcnc/apps/drywall_cutter_app/widget_xy_move_drywall.py

```diff
@@ -179,9 +229,12 @@ def __init__(self, **kwargs):
         super(XYMoveDrywall, self).__init__(**kwargs)
         self.m=kwargs['machine']
         self.sm=kwargs['screen_manager']
+        self.l=kwargs['localization']
 
         self.set_jog_speeds()
 
+        Clock.schedule_interval(self.check_zh_at_datum, 0.04)
```

Comment:
Avoid clock usage like this. Use event based approach that we discussed in sprint meet.